src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/storage/index.md

```diff
@@ -52,3 +52,27 @@ If you manage your own cluster and don’t have offerings from cloud providers a
 - [Longhorn](https://longhorn.io/)
 - [Rook (CephFS)](https://rook.io/)
 - [GlusterFS](https://www.gluster.org/)
+
+## Migrating from NFS storage to a custom StorageClass
+
+If you installed the Kubernetes agent using the default NFS storage, and want to use a custom `StorageClass` instead, simply rerun the installation Helm command with specified values for `persistence.storageClassName` and `persistence.size`.
+
+The following steps assume your Kubernetes agent is in the `octopus-agent-nfs-to-pv` namespace:
+
+1. Take note of the current Helm release and Chart version for your Kubernetes agent by running the following command:
+   ```bash
+   helm list --namespace octopus-agent-nfs-to-pv
+   ```
+1. The output should look like this:
+   - In this example, the release name is `nfs-to-pv` while the chart version is `1.0.1`
+   :::figure
+   ![Helm list command](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/kubernetes-agent-helm-list.png)
+   :::
+1. Run the following command (substitute with your own values):
```

Comment:
This part looks a bit busy, any suggestions?

Reply:
There is a code block component that might make this easier to layout?

Example
https://octopus.com/docs/deployments/custom-scripts#scripts-that-block-deployments

Reply:
I split the steps out into headings - which made it nicer